recipe/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
```

Comment:
Yep, there are a few other flags that should be set (similar to the build.sh script). I just wanted to see something working on AppVeyor before going into those other details... but the AppVeyor jobs have been queued for the last hour or so.